.github/workflows/nox.yml

```diff
@@ -34,24 +34,10 @@ jobs:
       env:
         version: ${{ matrix.python-version }}
     - name: Install dependencies
-      run: pip install nox
+      run: pip install nox[uv]
     - name: Test with nox
       run: nox -e "pytest-${{ env.PY_VERSION }}"
-    - name: Commit notebooks
```

Comment:
On https://github.com/microsoft/azure-quantum-tgp/settings/actions enable "Read and write permissions" to have this action work still.

Reply:
Too bad. Should remove this line then: https://github.com/microsoft/azure-quantum-tgp/blob/main/README.md?plain=1#L15

Reply:
Removed